alerts/openshift-container-storage-operator/CephOSDDown.md

```diff
@@ -0,0 +1,22 @@
+# CephOSDDown
+
+## Meaning
+
+OSD (Object Storage Daemon) are down indicates ceph-osd daemons not running as expected.
```

Comment:
This could also impact data availability or lead to data loss if too many OSDs are down. Or if the PGs are healthy, this is not a health issue and the cluster would be just fine even with an OSD down. Let's get PM and support input on this alert documentation.

Reply:
@agarwal-mudit @Lijo.Thomas1@ibm.com can you provide your thoughts on this alert.